crates/ezpdb/Cargo.toml

```diff
@@ -1,19 +1,19 @@
-[package]
-name = "ezpdb"
-version = "0.4.0"
-authors = ["Lander Brandt <landerbrandt@gmail.com>"]
-edition = "2018"
-license = "MIT"
-readme = "README.md"
-repository = "https://github.com/landaire/pdbview"
-description = "High-level library for https://crates.io/crates/pdb"
-
-[dependencies]
-pdb = "0.7"
-thiserror = "1.0"
-log = "0.4"
-serde = { version = "1.0", features = ['derive', 'rc'], optional = true }
-uuid = "0.8"
-
-[patch.crates-io]
-pdb = { git = "https://github.com/landaire/pdb"}
+[package]

```

Comment:
This looks like just a whitespace change? `LF` -> `CRLF`? Could you remove this from the PR if so?

Reply:
Thanks, reverted this.